From 1be1c2bc51b17ee00cd58115e6c8770c6f530649 Mon Sep 17 00:00:00 2001
From: flar2 <asegaert@gmail.com>
Date: Mon, 30 Dec 2013 00:42:17 -0500
Subject: [PATCH 249/276] Revert "Enable UNALIGNED_ACCESS for the new LZO
 algorithm to use. LZO decrompression is used on
 QCACHE so we benefit a little from the new and
 improved LZO code."

This reverts commit 4dc33c808cb1c2e9108382d00110927b0c841afd.

Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 arch/Kconfig |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/Kconfig b/arch/Kconfig
index 63fc37a..0a3ffe4 100644
--- a/arch/Kconfig
+++ b/arch/Kconfig
@@ -78,7 +78,6 @@ config OPTPROBES
 
 config HAVE_EFFICIENT_UNALIGNED_ACCESS
 	bool
-	def_bool y
 	help
 	  Some architectures are unable to perform unaligned accesses
 	  without the use of get_unaligned/put_unaligned. Others are
-- 
1.7.9.5

